src/styles/main.scss

```diff
@@ -115,6 +115,13 @@ header.navbar .navbar-brand {
   display: block !important;
 }
 
+.cke_combo_button {
+  .cke_combo_text {
+    width: calc(100% - 29px) !important;
```

Comment:
Usage of `!important` and absolute values is a bad practice.

The following code can be used to fix the issue:
```
#editor .cke_combo_button {
  .cke_combo_text {
    width: auto;
  }
}
```

Reply:
But I would be limiting possible width by `max-width` to prevent very long buttons

Reply:
:heavy_check_mark: